astropy/table/pprint.py

```diff
@@ -13,19 +13,26 @@
 import numpy as np
 
 from .. import log
-# Note, in numpy <= 1.6, some classes do not properly represent themselves.
 from ..utils.console import Getch, color_print, terminal_size, conf
 from ..utils.data_info import dtype_info_name
 
-if six.PY3:
+__all__ = []
```

Comment:
It should be ok for `table.pprint` as it seems not imported/documented, but yeah, better safe than sorry. I let you judge on the final choice, it was a detail.